valuesets/ValueSet-UKCore-SpecimenCondition.xml

```diff
@@ -0,0 +1,29 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+	<id value="UKCore-BiopsyState"/>
+	<url value="https://fhir.hl7.org.uk/ValueSet/UKCore-BiopsyState"/>
+	<version value="2.1.0"/>
+	<name value="UKCoreBiopsyState"/>
+	<title value="UK Core Biopsy State"/>
+	<status value="active"/>
+	<date value="2023-04-28"/>
+	<publisher value="HL7 UK"/>
+	<contact>
+		<name value="HL7 UK"/>
+		<telecom>
+			<system value="email"/>
+			<value value="ukcore@hl7.org.uk"/>
+			<use value="work"/>
+			<rank value="1"/>
+		</telecom>
+	</contact>
+	<description value="A set of codes to record how has the sample been preserved/fixed."/>
+	<copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html."/>
+		<compose>
+        <include>
+            <system value="http://terminology.hl7.org/CodeSystem/v2-0493" />
+        </include>
+		<include>
+			<system value="https://fhir.hl7.org.uk/CodeSystem/UKCore-BiopsyState"/>
+		</include>
+    </compose>
```

Comment:
Should we have a manual expansion?

Reply:
I was going to get it in and do a proper expansion using swagger as the amount of times Ive messed up trying to add them manually. We need to come up with a better system

Reply:
Yeah, we definitely do!